packages/sil-voting-app/src/components/Header.tsx

```diff
@@ -0,0 +1,34 @@
+import { Theme } from '@material-ui/core';
+import { makeStyles } from '@material-ui/styles';
+import Block from 'medulas-react-components/lib/components/Block';
+import CircleImage from 'medulas-react-components/lib/components/Image/CircleImage';
+import Typography from 'medulas-react-components/lib/components/Typography';
+import React from 'react';
+import icon from '../assets/iov-logo.svg';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  logo: {
+    flexBasis: '205px',
+    '& > div + h5': {
```

Comment:
What is this?

Reply:
It's a way to specify the gaps in the container element instead of on each element, but in this case I can apply it directly to the `h5` to make it more readable.

Reply:
Yes, it is better if you assign directly a new rule directly to those elements, rather than using selectors on parents elements.

Does it complicate your solution?

Reply:
It's okay, I wrapped Typography in a Block with marginLeft.